omero/sysadmins/repository-move.txt

```diff
@@ -117,3 +117,37 @@ to ``/Volumes/imports/ManagedRepository``:
     If :property:`omero.managed.dir` is set then the Managed Repository and
     the OMERO data directory should be treated independently and thus be moved
     separately if necessary.
+
+Extending the Managed Repository
+--------------------------------
+
+It is possible to leave the Managed Repository in place yet have newly
+imported image files stored on a different underlying storage volume.
+For example, if your :property:`omero.managed.dir` is set to
+:file:`/mnt/omero/ManagedRepository` and, as it fills, it would be
+better for future images to be imported onto
+:file:`/mnt/omero/large-volume-B/` then an OMERO administrator may use
+the :omerocmd:`fs mkdir` subcommand to properly set up a subdirectory in
+the existing Managed Repository:
+::
+
+    bin/omero fs mkdir volume-B
```

Comment:
I was thinking that by the different naming it helps the user to see the difference between the two directories. Though even if you don't symlink you can probably mount the volume at `large-volume-B` also at `volume-B`. I'll look at rewording anyway though, probably clearer if I separate as we *really* don't want readers being confused at this point.